jvb/src/main/kotlin/org/jitsi/videobridge/xmpp/XmppConnection.kt

```diff
@@ -52,7 +54,7 @@ public class XmppConnection : IQListener {
 
     var eventHandler: EventHandler? = null
 
-    fun start() {
+    fun start(statsManager: StatsManager?) {
```

Comment:
I think I prefer this the way it was before: `XmppConnection` is just a transport which doesn't need to care about what it's carrying.  A layer above decides we want to transmit stats over this transport and creates an appropriate stats transport (`MucStatsTransport`) and passes it the transport to use (the `XmppConnection`) without `XmppConnection` knowing or caring.

Reply:
OK. My concern was that `Main` was becoming a bit of a mess. I'll think about it and refactor in the morning (same applies to the one for callstats)